packages/synapse-react-client/src/components/GenericCard/GenericCard.tsx

```diff
@@ -688,8 +692,15 @@ export default class GenericCard extends React.Component<
       let value: any = data[schema[columnName]]
       let columnDisplayName
       if (value) {
-        // SWC-6115: special rendering of the version column (for Views)
-        if (isView && columnName === 'currentVersion') {
+        // PORTALS-2750: special rendering of the datasetSizeInBytes (for Dataset Collections)
+        if (
+          isDatasetCollection(table as Entity) &&
+          columnName === 'datasetSizeInBytes'
+        ) {
+          columnDisplayName = 'Size'
+          value = calculateFriendlyFileSize(parseInt(value))
```

Comment:
If this is a Dataset Collection and the column is named "datasetSizeInBytes", then use a special renderer.  Output the display column "Size" and a friendly file size